### ISSUE TITLE
Google Workspace Profile Image wrong URL (Error 400)

### ISSUE THREAD

**Issue by AceScottie** [grokability/snipe-it#14351]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

When logging in using Google Workspace the profile image URL is incorrect resulting in a missing profile image.
wrong_url: `https://lh3.googleusercontent.com/a-/{data}`
correct_url: `https://lh3.googleusercontent.com/fife/{data}`

### Reproduction steps

1.Setup Google Workspace Login
2.Login with valid google workspace user
3.check profile image

### Expected behavior

Epected to see correct Profile image when logging in.

### Screenshots

Default:
![missing_profile_image](https://github.com/snipe/snipe-it/assets/7810499/3f6f39a7-a307-4c7e-9b96-f9c9934cf6c1)

Manually Editing to add correct URL
![correct_profile_image](https://github.com/snipe/snipe-it/assets/7810499/b0e0fd5b-492e-4cff-b4bd-b0c522aca8ad)

### Snipe-IT Version

v6.2.4-pre - build 12343

### Operating System

Windows 10

### Web Server

Apache

### PHP Version

8.1

### Operating System

Windows

### Browser

Google Chrome

### Version

122.0.6261.95

**Comment by mauro-miatello** [comment]
it works for me with https://lh3.googleusercontent.com/a/{data} (only a, not dash)

**Comment by AceScottie** [comment]
I updated to 6.3.4 build 13139 and it still doesn't work.
The link generated is `https://lh3.googleusercontent.com/a-/ALV-UjVyM81VwI0icTMTmUZGbg5qW_38AqppjbhYZbg7SqKLQ9UPlBXCPEJqY8yz93PpyZ1xrZNoPJWJfoWtOwrtax3I1b5rxa9jJEDA9HaooSDojZ89xwebYg3aIpvhg8cd6EGJACadp5e1xztENLMbWW`
the html for this section is:
`<img src="http://inventory.mydomain.net/uploads/avatars/https://lh3.googleusercontent.com/a-/ALV-UjVyM81VwI0icTMTmUZGbg5qW_38AqppjbhYZbg7SqKLQ9UPlBXCPEJqY8yz93PpyZ1xrZNoPJWJfoWtOwrtax3I1b5rxa9jJEDA9HaooSDojZ89xwebYg3aIpvhg8cd6EGJACadp5e1xztENLMbWW" class="img-responsive">`~

I have deleted it and try again. And also deleted then added gravitar but when i login it defaults to the google version. 

I have tried replacing `/a-/` with `/a/` and `/fife/` and none of these work anymore.

Is there a setting or API in admin.google to allow these ?
